src/Recipe.js

```diff
@@ -0,0 +1,18 @@
+import style from './recipe.module.css'
+
+const Recipe = ({title, calories, image, ingredients}) => {
+    return(
+        <div className={style.recipe}>
+            <h1>{title}</h1>
+            <ol>
+                {ingredients.map(ingredient => (
+                    <li>{ingredient.text}</li>
+                ))}
+            </ol>
+            <p>{calories}</p>
+            <img className={style.image} src={image} alt="" />
```

Comment:
For images, your PL might ask you to provide an `alt` description to help make your project's application as accessible as possible.  

A possible addition here would be

```
<img className={style.image} src={image} alt={`${ingredient.text} ingredient`} />

```